plugins/woocommerce-blocks/tests/integration/active-filters.spec.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import { fireEvent, screen, within } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import { initializeEditor, selectBlock } from '../utils/integration/editor';
+import '../../assets/js/blocks/active-filters';
```

Comment:
Nitpick: I think we should replace relative imports with path aliases. For example:
```javascript
// Current Import
import { initializeEditor, selectBlock } from '../utils/integration/editor';

// Suggested Import
import { initializeEditor, selectBlock } from '@utils/editor';
```

I guess we will need to make changes to "moduleNameMapper" in `jest.integration.config.json` file. What do you think?